releasenotes/notes/0.7/fusion-threshold-33127b9b4e219640.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Updates gate fusion default thresholds so that gate fusion will be applied
+    to circuits with of more than 14 qubits for statevector simulations on the
+    :class:`~qiskit.providers.aer.StatevectorSimulator` and 
+    :class:`~qiskit.providers.aer.QasmSimulator`.
```

Comment:
I might have put this under upgrade instead of feature, just because it's a change that a user might need to know about and make a change around for upgrading from aer 0.6.x to 0.7.0. But it's  up to you since this mostly a performance improvement and I doubt people care if gate fusion is happening under the covers